pkg/internal/mock/didcomm/protocol/mock_didexchange.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package protocol
+
+import "github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+
+// MockDIDExchangeSrv mock did exchange service
+type MockDIDExchangeSrv struct {
```

Comment:
Srv is usually the abbreviation for Server.

Reply:
fixed